proto/vtctldata.proto

```diff
@@ -811,6 +813,60 @@ message InitShardPrimaryResponse {
   repeated logutil.Event events = 1;
 }
 
+message MoveTablesCreateRequest {
+  // The necessary info gets passed on to each primary tablet involved
+  // in the workflow via the CreateVRWorkflow tabletmanager RPC.
+  string workflow = 1;
+  string source_keyspace = 2;
+  string target_keyspace = 3;
+  repeated string cells = 4;
+  repeated string tablet_types = 5;
+  repeated string source_shards = 6;
+  bool all_tables = 7;
+  repeated string include_tables = 8;
+  repeated string exclude_tables = 9;
+  // ExternalCluster is the name of the mounted cluster which has the source keyspace/db for this workflow
+  // it is of the type <cluster_type.cluster_name>
+  string external_cluster = 10;
+  // SourceTimeZone is the time zone in which datetimes on the source were stored, provided as an option in MoveTable
+  string source_time_zone = 11;
+  // OnDdl specifies the action to be taken when a DDL is encountered.
+  string on_ddl = 12;
+  // StopAfterCopy specifies if vreplication should be stopped after copying.
+  bool stop_after_copy = 13;
+  // DropForeignKeys specifies if foreign key constraints should be elided on the target.
+  bool drop_foreign_keys = 14;
+  // DeferSecondaryKeys specifies if secondary keys should be created in one shot after table copy finishes.
+  bool defer_secondary_keys = 15;
+  // Start the workflow after creating it.
+  bool auto_start = 16;
+}
+
+message MoveTablesCompleteRequest {
```

Comment:
in general we try to keep request/response types next to each other (so `CreateRequest`, `CreateResponse`, `CompleteRequest`, and so on)